org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/xbase/XbaseGeneratorFragment2.xtend

```diff
@@ -56,6 +56,9 @@ class XbaseGeneratorFragment2 extends AbstractXtextGeneratorFragment {
 		if (!grammar.inheritsXbase)
 			return;
 		
+		if (!grammar.eclipsePluginEditor.equals(grammar.eclipsePluginXbaseEditor))
```

Comment:
please enclose single line evaluation into curly braces

Reply:
Added the curly braces.